imod/mf6/simulation.py

```diff
@@ -1331,51 +1331,88 @@ def mask_all_models(
     def from_imod5_data(
         cls,
         imod5_data: dict[str, dict[str, GridDataArray]],
-        period_data: dict[str, dict[str, GridDataArray]],
-        allocation_options: SimulationAllocationOptions,
-        distributing_options: SimulationDistributingOptions,
+        period_data: dict[str, list[datetime]],
         times: list[datetime],
-        regridder_types: dict[str, RegridMethodType] = {},
+        allocation_options: Optional[SimulationAllocationOptions] = None,
+        distributing_options: Optional[SimulationDistributingOptions] = None,
+        regridder_types: Optional[dict[str, RegridMethodType]] = None,
     ) -> "Modflow6Simulation":
         """
-        Imports a GroundwaterFlowModel (GWF) from the data in an IMOD5 project file.
-        It adds the packages for which import from imod5 is supported.
-        Some packages (like OC) must be added manually later.
-
+        Imports a GroundwaterFlowModel (GWF) from the data in an IMOD5 project
+        file and puts it in a simulation. It adds the packages for which import
+        from imod5 is supported. Some packages (like OC) must be added manually
```

Comment:
`It adds the packages for which import  from imod5 is supported.`
This sentence is a bit unclear/ could some more explaination.

Which packages are supported? Do some packages have a `from_imod5_data` method? How do i now which packages are imported?

Reply:
Thanks, I think we need some kind of list somewhere to add what iMOD5 functionality is supported and what is not yet.